web/text_layer_builder.css

```diff
@@ -17,7 +17,7 @@
   position: absolute;
   text-align: initial;
   inset: 0;
-  overflow: hidden;
+  overflow: clip;
```

Comment:
Using `clip` instead of `hidden` is only needed for Chrome and not for Firefox. When selecting text, Chrome was sometimes trying to move it closer to the center of the webview by scrolling the contents of `.textLayer`, misaligning it from the underlying canvas.

The difference between `hidden` and `clip` is that `hidden` still allows scrolling the element's contents through indirect user interactions (for example, when focusing on an element or through `.scrollTo()`), while `clip` completely disables scroll. For this reason, `clip` is also ok in Firefox so I didn't add a `MOZCENTRAL` check.